docs/tutorial.md

```diff
@@ -1,5 +1,910 @@
+                <p class="title"><em>Ocean </em>life is brutal</p>
+                <p class="author">Surphy McBrah</p>
+            </li>
+            <li>
+                <p class="title">
+                    Family friendly fun at the
+                    <em>ocean </em>exhibit
+                </p>
+                <p class="author">Guy Prosales</p>
+            </li>
+        </ul>
+    </div>
+
+    <div class="story">
+        <h1>Ocean life is brutal</h1>
+        <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+            aliquip ex ea commodo consequat.
+        </p>
+        <p class="signature">Surphy McBrah</p>
+    </div>
+    
+    <div class="autoupdate">
+        Auto update: <input type="checkbox" />
+    </div>
+</div>
+```
+
+...and some CSS [here](https://zaceno.github.com/hatut/style.css).
+
+It looks like this:
+
+![Initial static mockup](./tut1.png)
+
+We'll start by making Hyperapp render the HTML for us. Then we will
+add dynamic behavior to all the widgets, including text input and 
+dynamically fetching stories.
+
+First, let's begin with the traditional "Hello World!"
+
+Hello World
+------------------------------
+
+Create this html file:
+
+```html
+<!doctype html>
+<html>
+  <head>
+    <link rel="stylesheet" href="https://zaceno.github.com/hatut/style.css">
+    <script type="module">
+
+// -- IMPORTS --
+
+import {h, app} from "https://unpkg.com/hyperapp?module"
+
+
+        
+// -- ACTIONS --
+
+
+                
+// -- VIEWS ---
+
+        
+        
+// -- RUN --
+
+app({
+  node: document.getElementById("app"),
+  view: () => h("h1", {}, [
+      "Hello ", 
+      h("i", {}, "World!")
+  ])
+})
+    </script>
+  </head> 
+  <body>
+    <div id="app"></div>
+  </body>
+</html>
+```
+
+> The sections demarked by comments are not important. They're
+> simply suggestions to help you organize the code that we'll be
+> adding throughout the tutorial.
+
+Open it in a browser, and you'll be greeted with an optimistic **Hello _World!_**.
+
+View
+------------------------------------
+
+Lets step through what just happened.
+
+### Virtual Nodes
+
+Hyperapp exports the `app` and `h` functions.
+`h` is for creating _virtual nodes_, which is to say: plain javascript objects
+which _represent_ DOM nodes.
+
+The result of  
+
+```js
+h("h1", {}, [
+  "Hello ", 
+  h("i", {}, "World!")
+])
+```
+
+is a virtual node, representing
+
+```html
+<h1>
+  Hello 
+  <i>World!</i>
+</h1>
+```
+
+### Rendering to the DOM
+
+`app` is the function that runs our app. It is called with a single argument - an object
+which can take several properties. For now we're just concerned with `view` and `node. `
+
+Hyperapp calls the `view` function to find out how we want the DOM. We express it in the form of virtual nodes.
```

Comment:
> ...to find out how we want the DOM

The phrasing seems a little weird, and may not translate well to non-native English speakers.

What about:

> ..., which describes what DOM elements we want to render.